packages/data-stores/src/reader/context/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { createContext } from '@wordpress/element';
+import type { ReactNode } from 'react';
+
+export type UserType = object | null;
```

Comment:
These types are a bit vague. Should we use the existing `UserData` type somehow?